scenarios/iam_privesc_by_rollback/cheat_sheet_raynor.md

```diff
@@ -1,9 +1,9 @@
-`aws configure --profile raynor`
+`aws configure --profile Raynor`
 
-`aws iam list-attached-user-policies --user-name raynor --profile Raynor`
+`aws iam list-attached-user-policies --user-name Raynor --profile Raynor`
```

Comment:
I suspect the naming may have changed since this PR. It looks like the user name get's created with the workspace cgid here: https://github.com/RhinoSecurityLabs/cloudgoat/blob/master/scenarios/iam_privesc_by_rollback/terraform/iam.tf#L3.

May need to look into how the other cheat sheet's handle this just to make sure it's consistent.

Reply:
Looks like we just use <name of somekind goes here>,  so something like this would probably be best for now:
```
--user-name <raynor user name>
```